Quark_Sketch/prj.conf

```diff
@@ -18,5 +18,6 @@ CONFIG_GPIO_QMSI_AON=y
 CONFIG_GPIO_QMSI_AON_NAME="GPIO_AON_0"
 CONFIG_GPIO_QMSI_AON_PRI=2
 CONFIG_PWM=y
+CONFIG_PWM_QMSI_NUM_PORTS=4
```

Comment:
this actually fixes the issue with pwm only working on pin3.
I will remove and create a separate PR